src/main/java/org/corfudb/cmdlets/corfu_layout.java

```diff
@@ -153,6 +154,7 @@
             log.debug("Bootstrapping with layout {}", l);
             try {
                 if (router.getClient(LayoutClient.class).bootstrapLayout(l).get()) {
+                    router.getClient(ManagementClient.class).bootstrapManagement(l).get();
```

Comment:
Shouldn't you be checking the return value of this bootstrap call? Also, I don't think this belongs in corfu_layout, which is supposed to be for layout servers. Please provide a clojure script which does this work.